mm2src/lp_ordermatch.rs

```diff
@@ -1356,23 +1373,27 @@ impl MakerOrderBuilder {
         }
 
         let rel_vol = &self.max_base_vol * &self.price;
-        if rel_vol < min_vol {
+        if rel_vol < min_rel_amount {
             return Err(MakerOrderBuildError::RelVolTooLow {
                 actual: rel_vol,
-                threshold: min_vol,
+                threshold: min_rel_amount,
             });
         }
 
-        if self.min_base_vol < min_vol {
+        let min_base_vol = match self.min_base_vol {
```

Comment:
I guess, this expression can be replaced with
```rust
let min_base_vol = self.min_base_vol.unwrap_or_else(|| min_base_amount.clone());
```

Reply:
Yeah, it will be shorter, let me do it :slightly_smiling_face:

Reply:
Done, please review again.